backend/inspirehep/mailing/config.py

```diff
@@ -15,6 +15,8 @@
 CONFERENCES_CONFIRMATION_EMAIL_ADDRESS = "conf@inspirehep.net"
 """Email address for sending confirmation email for conferences"""
 
+SEMINARS_CONFIRMATION_EMAIL_ADDRESS = "seminars@inspirehep.net"
```

Comment:
do we even have this kind of emails?

Reply:
I am not sure, that was the requirement. but it will work either way right?

Reply:
see last messages on https://inspirehep.zulipchat.com/#narrow/stream/195298-experts/topic/virtual.20seminars.20in.20INSPIRE

Reply:
👍